pkg/tpm/tpm.go

```diff
@@ -127,24 +153,25 @@ func getInfo() (*Info, error) {
 		}
 	}
 
-	owned, err := strconv.ParseBool(string(ownedBytes))
+	owned, err := strconv.ParseBool(string(ownedBytes[0]))
```

Comment:
was this failing because of the newline?

Reply:
yes